gcp-observability/src/main/java/io/grpc/gcp/observability/GlobalLoggingTags.java

```diff
@@ -35,20 +33,16 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-/** A container of all global custom tags used for logging (for now). */
+/** A container of all global location tags used for observability. */
 final class GlobalLoggingTags {
```

Comment:
Since the class specifically contains "location" tags, should the class be called GlobalLocationTags? Or more types of tags to be added?

Reply:
Good point. Renamed everywhere.